### PR TITLE
Update GluonCV importer

### PR DESCRIPTION
**Pull request by frankfliu**
Change-Id: I7483461aab3975d5ab1f13ca5875cc9ea99e20b1

## Channel for questions ##
Before or while filing an issue please feel free to join our [Slack channel](https://join.slack.com/t/deepjavalibrary/shared_invite/enQtODAwOTg0OTUzNDEwLTdlY2M4NmNlZTMzM2Q4MDUyZThkYzQwNmE2MDVhNTRiZDcxNTNlMWJhNTZkNWE1OGU2Nzg3MmY1OWQzN2Q5Mzk) to get in touch with development team, ask questions, find out what's cooking and more!

## Description ##
(Brief description of what this PR is about)

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- [ ] Code is well-documented: 
    - For user-facing API changes, Java doc has been updated. 
    - For new examples, README.md is added to explain the what the example does.
- [ ] To the my best knowledge, [examples](https://github.com/awslabs/djl/tree/master/examples) and [jupyter notebooks](https://github.com/awslabs/djl/tree/master/jupyter) are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, Java doc)
- [ ] Feature2, tests, (and when applicable, Java doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

File: model-uploader/src/main/java/ai/djl/uploader/ExporterArguments.java [hunk 1]
```diff
@@ -94,16 +107,16 @@ public static Options getOptions() {
...
-                "an",
-                "artifact-name",
-                true,
-                "the name of the artifact or the name of the model to import");
         options.addRequiredOption(
                 "ai",
                 "artifact-id",
                 true,
                 "the way to categorize the artifact, for example, resnet50 has resnet as artifact id");
+        options.addOption(
```
Comment (lanking520) [inline model-uploader/src/main/java/ai/djl/uploader/ExporterArguments.java:115]: If user import Keras model without specify the name. Saying he has a file stored in:

`path/my_model.h5`

Then the output of the metadata will be `null.h5`.

File: model-uploader/src/main/java/ai/djl/uploader/MetadataBuilder.java [hunk 2]
```diff
@@ -451,6 +453,16 @@ private Artifact constructArtifact(Path destination) throws IOException {
             item.setUri(uri);
             files.put(names[0], item);
         }
+        if (application == Application.CV.IMAGE_CLASSIFICATION
```
Comment (lanking520) [inline model-uploader/src/main/java/ai/djl/uploader/MetadataBuilder.java:456]: Can you do this when you do gluoncv import. Even if they are using same dataset, they may not have the same synset.

File: model-uploader/src/main/java/ai/djl/uploader/ExporterArguments.java (hunk 1 above)
Comment (frankfliu) [reply model-uploader/src/main/java/ai/djl/uploader/ExporterArguments.java:115]: if artifact-name is not specified, it will be set to artifact-id.
user has to specify artifact-name if it's different from artifact-id

File: model-uploader/src/main/java/ai/djl/uploader/MetadataBuilder.java (hunk 2 above)
Comment (frankfliu) [reply model-uploader/src/main/java/ai/djl/uploader/MetadataBuilder.java:456]: all gluon cv image classification with imagenet has the same synset.
I didn't add synset for cifar10

Comment (lanking520) [reply model-uploader/src/main/java/ai/djl/uploader/MetadataBuilder.java:456]: > all gluon cv image classification with imagenet has the same synset.
> I didn't add synset for cifar10

But a keras model may have a different synet even if it is `imagenet`. synset is a platform specific class

Comment (frankfliu) [reply model-uploader/src/main/java/ai/djl/uploader/MetadataBuilder.java:456]: image classification usually requires "synset", we only add this if "synset" is missing
